data/assets/scene/solarsystem/missions/jwst/jwst.asset

```diff
@@ -0,0 +1,315 @@
+local sunTransforms = asset.require('scene/solarsystem/sun/transforms')
+
+asset.require("spice/base")
+asset.require('scene/solarsystem/missions/jwst/Lagrange_points')
+local transforms = asset.require('scene/solarsystem/missions/jwst/L2')
+
+local JWSTKernel = asset.syncedResource({
+    Name = "JWST Kernel",
+    Type = "HttpSynchronization",
+    Identifier = "jwst_kernels",
+    Version = 1
+})
+
+local JWSTHorizons = asset.syncedResource({
+    Name = "JWST Horizons",
+    Type = "HttpSynchronization",
+    Identifier = "jwst_horizons",
+    Version = 1
+})
+
+local model = asset.syncedResource({
+    Name = "JWST Model",
+    Type = "HttpSynchronization",
+    Identifier = "jwst_model",
+    Version = 1
+})
+
+local band = asset.syncedResource({
+    Name = "JWST band texture",
+    Type = "HttpSynchronization",
+    Identifier = "jwst_band_texture",
+    Version = 1
+})
+
+local JWSTPosition = {
+    Identifier = "JWSTPosition",
+    Parent = transforms.L2.Identifier,
+    Transform = {
+        Translation = {
+            Type = "SpiceTranslation",
+            Target = "JWST", 
+            Observer = "392", -- L2
+            Kernels = JWSTKernel .. "/jwst_horizons_20200101_20240101_v01.bsp"
+        },
+    },
+    GUI = {
+        Name = "JWST Position",
+        Path = "/Solar System/Missions/JWST",
+        Hidden = true,
+    }
+}
+
+local JWSTRotation = {
+    Identifier = "JWSTRotation",
+    Parent = JWSTPosition.Identifier,
+    Transform = {
+        Rotation = {
+            Type = "FixedRotation",
+            Attached = "JWSTRotation",
+            XAxis = { 1, 0, 0 },
+            XAxisOrthogonal = true,
+            YAxisInvert = true,
+            YAxis = sunTransforms.SolarSystemBarycenter.Identifier
+        }
+    },
+    GUI = {
+        Name = "JWST Rotation",
+        Path = "/Solar System/Missions/JWST",
+        Hidden = true,
+    }
+}
+
+local JWSTBand = {
+    Identifier = "JWSTBand",
+    Parent = JWSTPosition.Identifier,
+    Transform = {
+        Rotation = {
+            Type = "FixedRotation",
+            Attached = "JWSTBand",
+            XAxis = { 1, 0, 0 },
+            XAxisOrthogonal = true,
+            ZAxis = sunTransforms.SolarSystemBarycenter.Identifier
+        }
+    },
+    Renderable = {
+        Enabled = false,
+        Type = "RenderableSphere",
+        Texture = band .. "/JWST-band.png",
+        Size = 9.2E15,
+        Segments = 50,
+        DisableFadeInOut = true,
+        Orientation = "Inside",
+        Opacity = 0.05,
+        UseAdditiveBlending = true,
+    },
+    Tag = { "fov" },
+    GUI = {
+        Name = "JWST View Band",
+        Path = "/Solar System/Missions/JWST",
+        Hidden = true,
+    }
+}
+
+local JWSTModel = {
+    Identifier = "JWSTModel",
+    Parent = JWSTRotation.Identifier,
+    TimeFrame = {
+        Type = "TimeFrameInterval",
+        Start = "2020 JAN 01",
+        End = "2024 JAN 01"
+    },
+    Transform = {
+        Rotation = {
```

Comment:
I don't think this is necessary;  each scenegraphnode should get a StaticRotation by default which also defaults to 0,0,0